refactor: rename config file and fix readme unclarity issues (#136)
0xPolygonMiden/miden-client: README.md

miden-client account import <path-to-accounts-directory>/*.mac
```

> The client will import all account-related data generated by the node stored as `*.mac` files, (located by default on the `./accounts` subdirectory of the node's root directory), and insert them in the local store. You should now be able to see the accounts by doing:

Reviewer: What do you think about:
```suggestion
The client will then import all account-related data generated by the node (stored as `*.mac` files), and insert them in the local store. Notice that, by default, the node generates the account files on the `./accounts` subdirectory. You should now be able to see the accounts by doing:
```
Because the node binary can be executed from anywhere, we shouldn't assume it will be on the subdirectory based on the node's root (I believe it's relative to the node's `miden.toml` file?)

Author: Ah, then I think it's not relative to the node's miden.toml but relative to where the make_genesis file is located. Perhaps in order to avoid confusions we could just say it's the accounts directory that get's generated when you run `make-genesis` with the node. What do you think?

Reviewer: I agree, I think that sounds better and we should try to reduce what we say about the node's binary as much as possible anyway.

Author: done!
